component/background/RoomService.tsx

```diff
@@ -11,11 +12,16 @@ const config = {
 };
 firebase.initializeApp(config);
 
-class RoomService {
+const allRooms = firebase.database().ref("rooms");
+const allUsers = firebase.database().ref("users");
+
+const MESSAGE = "message";
+const USER = "user";
```

Comment:
What's the different?
These variables are not accessible anywhere else. After compiled static fields of a class and global variables are the same